packages/labs/router/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+export * from './router.js';
```

Comment:
Why have an `index.js` rather than just setting `main` to `router.js`?

Reply:
There's going to be other files soon...